Twilio/twilio_send_email.py

```diff
@@ -41,10 +42,10 @@ def sendgrid_verification_email(user_email, studyhub_code) -> bool:
         sg = SendGridAPIClient(twilio_api_key)
         response = sg.send(message)
         return (
-            True,
+            HTTPStatus.CREATED,
```

Comment:
good use of constant. since you used constant here, might as well do it in all other place that have hard coded status code.